airflow/config_templates/default_test.cfg

```diff
@@ -64,6 +64,7 @@ auth_backend = airflow.api.auth.backend.default
 
 [operators]
 default_owner = airflow
+default_queue = default
```

Comment:
```suggestion
```

We don't need to include defaults here if it's the same default from airflow.cfg

Reply:
Removed as suggested, this applied to the old config value under the celery section (it was set to the same value as airflow.cfg)

Reply:
Yeah, removing makes sense, just no need to add it to the new section